modules/boulder/src/main/ts/ephox/boulder/alien/SimpleResult.ts

```diff
@@ -3,7 +3,10 @@ import { Arr, Result } from '@ephox/katamari';
 // An experiment to make a more efficient boulder.
 export type SimpleResult<E, A> = SimpleError<E> | SimpleValue<A>;
 
-export enum SimpleResultType { Error, Value }
+export enum SimpleResultType {
```

Comment:
Is the rule that is there is just one field in the object it should be on one line, but otherwise multiple? Or is it more nuanced than that?

Reply:
The existing linter rule just makes it so it's always defined as a block by the look. It now just checks types/interfaces as well. See https://eslint.org/docs/latest/rules/brace-style